test/realizations/Formulation_Manager_Test.cpp

```diff
@@ -248,49 +230,28 @@ const std::string EXAMPLE_1 = "{ "
         "} "
     "} "
 "}";
-
 const std::string EXAMPLE_2 = "{ "
     "\"global\": { "
       "\"formulations\": [ "
-        "{"
-          "\"name\": \"tshirt\", "
-          "\"params\": {"
-            "\"maxsmc\": 0.81, "
-            "\"wltsmc\": 1.0, "
-            "\"satdk\": 0.48, "
-            "\"satpsi\": 0.1, "
-            "\"slope\": 0.58, "
-            "\"scaled_distribution_fn_shape_parameter\": 1.3, "
-            "\"multiplier\": 1.0, "
-            "\"alpha_fc\": 1.0, "
-            "\"Klf\": 0.0000672, "
-            "\"Kn\": 0.1, "
-            "\"nash_n\": 8, "
-            "\"Cgw\": 1.08, "
-            "\"expon\": 6.0, "
-            "\"max_groundwater_storage_meters\": 16.0, "
-            "\"nash_storage\": [ "
-                "1.0, "
-                "1.0, "
-                "1.0, "
-                "1.0, "
-                "1.0, "
-                "1.0, "
-                "1.0, "
-                "1.0 "
-            "], "
-            "\"soil_storage_percentage\": 1.0, "
-            "\"groundwater_storage_percentage\": 1.0, "
-            "\"timestep\": 3600, "
-            "\"giuh\": { "
-                "\"giuh_path\": \"./test/data/giuh/GIUH.json\", "
-                "\"crosswalk_path\": \"./data/crosswalk.json\" "
+            "{"
+              "\"name\":\"bmi_c++\","
+              "\"params\": {"
+                "\"model_type_name\": \"test_bmi_cpp\","
+                "\"library_file\": \"{{EXTERN_DIR_PATH}}/test_bmi_cpp/cmake_build/libtestbmicppmodel.so\","
+                "\"init_config\": \"{{BMI_C_INIT_DIR_PATH}}/test_bmi_c_config_0.txt\","
+                "\"main_output_variable\": \"OUTPUT_VAR_2\","
+                "\"" BMI_REALIZATION_CFG_PARAM_OPT__VAR_STD_NAMES "\": { "
+                  "\"INPUT_VAR_2\": \"" AORC_FIELD_NAME_TEMP_2M_AG  "\","
+                  "\"INPUT_VAR_1\": \"" AORC_FIELD_NAME_PRECIP_RATE "\""
+                "},"
+                "\"create_function\": \"bmi_model_create\","
+                "\"destroy_function\": \"bmi_model_destroy\","
+                "\"uses_forcing_file\": false"
+              "} "
             "} "
-          "} "
-        "} "
       "], "
       "\"forcing\": { "
-          "\"file_pattern\": \".*{{ID}}.*.csv\", "
+          "\"file_pattern\": \".*{{ID}}.*.csv\", " // DIFF from Ex.1
```

Comment:
Are these comments placed to get `git diff` to behave sensibly? That's the only possibility I can infer from them

Reply:
Ah... those are I suppose vestiges from a676c5d, where I was trying to capture how the examples differed from one another, to determine what the tests were actually testing and hence what needed to be preserved. Still somewhat useful as road markers, since the four giant strings aren't easily comparable. (I recommend cutting example 1 and pasting it after example 4 if you want, that's how I compared them to get those notes.)